examples/medplum-patient-intake-demo/src/bots/core/intake-utils.ts

```diff
@@ -504,6 +505,42 @@ export async function addImmunization(
   );
 }
 
+/**
+ * Adds a CareTeam resource associating the patient with a pharmacy
+ *
+ * @param medplum - The Medplum client
+ * @param patient - The patient beneficiary of the care team
+ * @param pharmacy - The pharmacy to be added to the care team
+ */
+export async function addPharmacy(
+  medplum: MedplumClient,
+  patient: Patient,
+  pharmacy: Reference<Organization>
+): Promise<void> {
+  await medplum.upsertResource(
+    {
+      resourceType: 'CareTeam',
+      status: 'proposed',
+      name: 'Patient Preferred Pharmacy',
+      subject: createReference(patient),
+      participant: [
+        {
+          member: pharmacy,
+          role: [{ coding: [{ system: SNOMED, code: '76166008', display: 'Practical aid (pharmacy) (occupation)' }] }],
+        },
+        {
+          member: createReference(patient),
```

Comment:
Remove the reference to the paitent as a "participant". Typically, the participants are just the clinicians involved

Reply:
Oh, got it. Will do! Thanks